src/common/constant.ts

```diff
@@ -225,6 +225,8 @@ export const QUICKSIGHT_USER_NAME_PATTERN =
   '^[A-Za-z0-9][A-Za-z0-9/_@.\\-]+[A-Za-z0-9]$';
 export const QUICKSIGHT_NAMESPACE_PATTERN = '^([A-Za-z])[A-Za-z0-9]{4,63}$';
 export const REDSHIFT_DB_NAME_PATTERN = `^${PROJECT_ID_PATTERN}$`;
+export const REDSHIFT_DB_USER_NAME_PATTERN = '^([a-z0-9-]{1,63})?$';
```

Comment:
We should allow the undersscore at least.

> Begin with an ASCII single-byte alphabetic character or underscore character, or a UTF-8 multibyte character two to four bytes long.

> Subsequent characters can be ASCII single-byte alphanumeric characters, underscores, or dollar signs, or UTF-8 multibyte characters two to four bytes long.

https://docs.aws.amazon.com/redshift/latest/dg/r_names.html#r_names-standard-identifiers